.travis.yml

```diff
@@ -109,8 +109,8 @@ script:
   - if [[ $DEVITO_BACKEND != 'yask' ]]; then python examples/checkpointing/checkpointing_example.py; fi
 
   # Test tutorial notebooks for the website using nbval
-  - if [[ $DEVITO_BACKEND != 'yask' ]]; then py.test -vs --nbval examples/seismic/tutorials; fi
-  - if [[ $DEVITO_BACKEND != 'yask' ]]; then py.test -vs --nbval examples/cfd; fi
+  - if [[ $DEVITO_BACKEND != 'yask' ]]; then DEVITO_OPENMP=1 OMP_NUM_THREADS=4 py.test -vs --nbval examples/cfd; fi
```

Comment:
I don't think we should enforce OPENMP in **all** builds by exporting DEVITO_OPENMP locally

Reply:
Was only trying to get around travis issues. Revered in e5d17f24